f2/src/components/formik/button/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { Button, Container, Row } from 'react-bootstrap'
+import { GoChevronRight } from 'react-icons/go'
+import { Route } from 'react-router-dom'
+import { FiPaperclip } from 'react-icons/fi'
+
+export const SubmitButton = (props) => {
+  return (
+    <Button type="submit" className="button submit-button">
+      <span className="button-label">
+        {props.label}
+        <GoChevronRight className="button-icon-right" />
+      </span>
+    </Button>
+  )
+}
+
+export const CancelButton = (props) => {
+  return (
+    <Route
+      render={({ history }) => (
+        <Button
+          type="button"
+          className="button cancel-button"
+          to="/confirmCancel"
```

Comment:
React-bootstrap probably doesn't know React Route well. I am wondering what's the use of to here.
Since you have already onclick to do a history push

Reply:
I just copied the pops from the Chakra-UI cancel button. You are right, I tested without the **to** prop and it works. I have updated the component.